documentation/announcing-az-module.md

```diff
@@ -0,0 +1,163 @@
+# Announcing New Module 'Az'
+In August 2018 we released a new module, 'Az' which combines the functionality of the AzureRM and AzureRM.Netcore modules.  Az runs on both PowerShell 5.1 and PowerShell Core.  'Az' ensures that the PowerShell and PowerShell core cmdlets for managing Azure resources will always be in sync and up to date.  In addition, Az will simplify and regularize the naming of Azure cmdlets, and the organization of Azure modules.  Az is intended as a replacement for the AzureRM.Netcore and AzureRM modules.
```

Comment:
@markcowl nit: there's one instance of PowerShell _core_ that probably should be PowerShell _Core_